test/checked.jl

```diff
@@ -166,6 +166,7 @@ import Base: checked_abs, checked_neg, checked_add, checked_sub, checked_mul,
     @test checked_cld(typemin(T), T(1)) === typemin(T)
     @test_throws DivideError checked_cld(typemin(T), T(0))
     @test_throws DivideError checked_cld(typemin(T), T(-1))
+
```

Comment:
I think you can revert the changes to this file now.

Reply:
done